### ISSUE TITLE
Subscribing to packages

### ISSUE THREAD

**Issue by CJ-Wright** [regro/cf-scripts#18]
@jakirkham commented on [Thu Mar 01 2018](https://github.com/regro/cf-graph/issues/49)

Not sure how you are checking for updates currently, but you may find issue ( https://github.com/pypa/warehouse/issues/1683 ) interesting. Basically asking PyPI to include some sort of feed for subscribing to specific packages for updates.

---

@CJ-Wright commented on [Fri Mar 02 2018](https://github.com/regro/cf-graph/issues/49#issuecomment-370074426)

That would be very cool.
Currently we troll through all of the packages looking at their upstream URLs, but having all the pypi packages managed through their own stream would be great!

---

@jakirkham commented on [Fri Mar 02 2018](https://github.com/regro/cf-graph/issues/49#issuecomment-370084884)

By upstream URLs do you mean `home`, `dev_url`, or something else?

---

@CJ-Wright commented on [Fri Mar 02 2018](https://github.com/regro/cf-graph/issues/49#issuecomment-370087717)

Wherever the `meta.yaml` describes `url` to be.

**Comment by jakirkham** [comment]
FYI PyPI has since implemented the package update feed.

xref: https://github.com/pypa/warehouse/issues/1683#issuecomment-622518230

**Comment by beckermr** [comment]
We're running on paperclips, rubber bands, and ☕ here. I am not confident this will happen any time soon. ;p

**Comment by jakirkham** [comment]
Of course. This was more "Hey, I got a notification from this issue we opened a while ago saying it's done. Let's share that with the team." 🙂
